.vsts-pipelines/variables/common.yml

```diff
@@ -0,0 +1,3 @@
+variables:
+- ${{ if ne(variables['System.TeamProject'], 'public') }}:
+  - template: common-official.yml
```

Comment:
I feel like separating the common-official variables isn't worthwhile.  The main detractor is that they contain groups which means you aren't going to be able to use the short hand syntax here if variables are ever added.  Having the common-official variables here would make that more obvious.